[WIP]: Notifications

JohnUUU opened the pull request.

JohnUUU committed:
changes so I can make PR

tianrendong commented on internal/twilio/twilio.go:
```diff
@@ -0,0 +1,28 @@
+	if err != nil {
+		fmt.Println("Error loading .env file")
+	}
+
```
@JohnUUU please make a note about env vars

JohnUUU replied:
Note: Env Vars store the Twilio Account SID and password, as well as the phone number that sends the notification. When we actually roll out the notification we probably want to change these to a department approved Twilio account.

JohnUUU committed:
Fix Notifications

- Notifications are working now, just had to also send the region over in the frontend and use that to decode the number

- Also expanded the message body to include stuff like the Course that sent the message and the timestamp at which the message was sent
